### ISSUE TITLE
Single car shortcode

### ISSUE THREAD

**Issue by barrykooij** [barrykooij/wp-car-manager#64]
Add a shortcode to display a single car.

**Comment by juewelsO** [comment]
Yes something like this shortcode would be perfect for our Website.
[wpcm_cars make="ford"]

**Comment by barrykooij** [comment]
@juewelsO This is actually not what this issue is about but has been added to the 1.2.0 update already. You can now do `[wpcm_cars make="tesla"]` for example and this will only output Tesla cars.

This issue is about being able to output a single car

**Comment by barrykooij** [comment]
Pushing this back because I'm not sure we'd like to implement this. All cars already have a detail page, why would you like to output that exact data on another page.
